script.js

```diff
@@ -0,0 +1,145 @@
+    cardTitle: 'Collaboration',
+    cardDetail: 'You will get a chance to collaborate with singers.',
+  },
+  {
+    image: 'images/videoconference.png',
+    cardTitle: 'Meetup',
+    cardDetail: 'You will get a chance to meet with your favoratie singers.',
+  },
+];
+
+const card = document.querySelector('#allcardS');
+
+for (let i = 0; i < projectCards.length; i += 1) {
+  card.innerHTML += `<div class="Card">
+                          <img src="${projectCards[i].image}" alt="card-icon">
+                          <h5>${projectCards[i].cardTitle}</h5>
+                         <p>
+                             ${projectCards[i].cardDetail}
+                         </p>
+                        </div>`;
+}
+
+const singers = [
+  {
+    singerImage: 'images/neha.jpg',
+    singerName: 'Neha Singh',
+    singerTitle: 'Neha is very good singer he have almost released 300 songs',
+    singerDetail: 'Neha has relased more than 300 songs, and he only sing in hindi language.',
+  },
+  {
+    singerImage: 'images/diljeet.jpeg',
+    singerName: 'Arjit Singh',
+    singerTitle: 'He is very good singer he have almost released 300 songs',
+    singerDetail: 'Benkler has relased more than 300 songs, and he only sing in hindi language.',
+  },
+  {
+    singerImage: 'images/arjit.jpeg',
+    singerName: 'Daljeet Singh',
+    singerTitle: 'He is very good singer he have almost released 300 songs',
+    singerDetail: 'Daljeet has relased more than 300 songs, and he only sing in hindi language.',
+  },
+  {
+    singerImage: 'images/sunidhi.jpg',
+    singerName: 'sunidhi Singh',
+    singerTitle: 'sunidhi is very good singer he have almost released 300 songs',
+    singerDetail: 'Sunidhi has relased more than 300 songs, and he only sing in hindi language.',
+  },
+  {
+    singerImage: 'images/shreya.jpg',
+    singerName: 'Shreya Singh',
+    singerTitle: 'shreya is very good singer he have almost released 300 songs',
+    singerDetail: 'shreya has relased more than 300 songs, and he only sing in hindi language.',
+  },
+  {
+    singerImage: 'images/sonu.jpg',
+    singerName: 'sonu nigham',
+    singerTitle: 'He is very good singer he have almost released 300 songs',
+    singerDetail: 'sonu has relased more than 300 songs, and he only sing in hindi language.',
+  },
+];
+
+function showMoreCards() {
+  const guest = document.getElementById('guest-list');
+  const seeMoreBtn = document.getElementById('seeMore');
+
+  for (let i = 2; i < singers.length; i += 1) {
+    guest.innerHTML += `<div class="singer">
+                            <img src="${singers[i].singerImage}" alt="card-singer-image">
+                            <div class="singer-details">
+                               <h3>${singers[i].singerName}</h3>
+                               <h4>${singers[i].singerTitle}</h4>
+                               <p>${singers[i].singerDetail}</p>
+                            </div>
+                         </div>`;
```

Comment:
I would recommend making each singerCard an `article` instead of `div`. That would make more sense semantically and search engines would thank you.

Reply:
- [ Optional ] Hello @muneebulrehman As suggested b y the previous reviewer for the purpose of using the correct semantic, Kindly I recommend you to use` article` instead of `div `tag for the project card.